### DISCUSSION TITLE
Enhanced Security and Improved Anime Quality

### DISCUSSION THREAD

**Discussion by alvarorichard** [alvarorichard/GoAnime#19] [General]
We are excited to announce the latest release of GoAnime! This update brings significant improvements to your anime-watching experience, including enhanced security measures and improved video quality.

**What's New in This Release:**

- **Security Enhancement:** We have implemented mitigation for path traversal vulnerabilities, ensuring a safer environment for our users.
- **Improved Anime Quality:** Enjoy your favorite anime with better video quality, providing a more immersive viewing experience.

We always strive to provide our users with the best possible experience. Your feedback is invaluable to us, so please let us know if you have any suggestions or encounter any issues.

Thank you for choosing GoAnime!

<hr /><em>This discussion was created from the release <a href='https://github.com/alvarorichard/GoAnime/releases/tag/v1.0.1'>Enhanced Security and Improved Anime Quality</a>.</em>

**Comment by Valdir2373** [comment]
muito bom, very good

**Comment by uruma-yxz** [comment]
Amei continua :3
